instance/models/mixins/openedx_config.py

```diff
@@ -390,6 +391,21 @@ def _get_configuration_variables(self):
                 ],
             })
 
+        if self.privacy_policy_url:
```

Comment:
Since we're adding TOS and privacy, you'll need to update this.
BTW, kudos on making the configuration only set these variables when the fields are not blank. :+1: